### PR TITLE
Porting b12 suit sensor selection menu and give code

### PR DESCRIPTION
**Pull request by ghost**
Will I ever finish holsters?

Someone figure out what's wrong with give.dm's code I can't into debugging even if it's probably obvious as fuck.

### CODE REVIEW

File: code/modules/clothing/clothing.dm [hunk 1]
```diff
@@ -260,38 +260,61 @@ BLIND     // can't see anything
 	if(hastie)
 		usr << "\A [hastie] is attached to it."
 
-atom/proc/generate_uniform(index,t_color)
+atom/proc/generate_uniform(index,t_color) //this fucking shit i swear to god ninja code
```
Comment (RemieRichards) [inline code/modules/clothing/clothing.dm]: Remove this comment.

I am 90% certain this code is what alters uniforms with no female sprite to fit the human female sprite.

Meaning it has NOTHING to do with ninja code.

**Comment by RemieRichards** [comment]
Paste the errors you get from give.dm

**Comment by ghost** [comment]
Thanks for fixing everything as usual @RemieRichards 

This is working now.

**Comment by ghost** [comment]
░░▒▒▒▒▒▒▒▒▒▒▒▒▒▒░░
░▒▒█░░▒▒▒▒██░░▒▒▒░
▒▒████▒▒▒█████▒▒▒▒
▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒
▒▒▒▒░Merge it now░▒▒▒▒
░▒▒░░░░░░░░░░░░▒▒░
░░▒▒░░░▓▓▓▓▓▓▒▒▒░░
░░░▒▒░▓▓▓▓▓▓▒▒▒░░░
░░░░░▒▒▒▒▒▒▒▒░░░░░
░░░░░░Cuz It's░░░░░░░
░█▀▀ ░█▀█ ░█ ░█▀▀ ░░
░█▀▀ ░█▀▀ ░█ ░█ ░░░░
░▀▀▀ ░▀ ░░░▀ ░▀▀▀░░░

**Pull request merged**